sys/net/gnrc/network_layer/ipv6/nib/_nib-slaac.c

```diff
@@ -34,6 +37,15 @@ void _auto_configure_addr(gnrc_netif_t *netif, const ipv6_addr_t *pfx,
     int idx;
     uint8_t flags = GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_TENTATIVE;
 
+#if !GNRC_IPV6_NIB_CONF_SLAAC
+    if (!gnrc_netif_is_6ln(netif)) {
+        LOG_WARNING("SLAAC not activated; won't auto-configure IPv6 for "
+                         "interface %u\n"
+                    "    Use " NAME(GNRC_IPV6_NIB_CONF_SLAAC) "=1 to "
```

Comment:
Why wouldn't a simple
```c
"    Use GNRC_IPV6_NIB_CONF_SLAAC=1 to "
```
suffice? The stringification of a macro to print the macro name seems unnecessarily complex in this situation. Or are there other intentions (for future use)?

Reply:
My main motivation is future-proofness, e.g. if the name of the macro changes. The example will cause an immediate compile-time error.

Reply:
>The example will cause an immediate compile-time error while just having a string literal won't.

Did you test that? I also thought so and did a quick test using a `puts()`. I was able to put anything in `NAME()` without the compiler throwing an error.

Reply:
No, I didn't I assumed the preprocessor checks what it is provided with. Ok then I do it the simple way ;-).